python/tempo/__init__.py

```diff
@@ -1,2 +1,14 @@
+from IPython import get_ipython
+
 from tempo.tsdf import TSDF
-from tempo.utils import display
\ No newline at end of file
+if (
+            ('create_dlt_table_fn' not in list(get_ipython().user_ns.keys()))
+            and
+            ('dlt_sql_fn' not in list(get_ipython().user_ns.keys()))
```

Comment:
You shouldn't need to construct this list twice. How about:

```
user_ns_labels = list(get_ipython().user_ns.keys())

if ( ('create_dlt_table_fn' not in user_ns_labels) and ('dlt_sql_fn' not in user_ns_labels))
[...]
```

Reply:
I agree I will make the change @tnixon

Reply:
pushed the changes